benchmarks/distributed_tree_driver/distributed_tree_driver.cpp

```diff
@@ -349,20 +351,58 @@ int main_(std::vector<std::string> const &args, const MPI_Comm comm)
     }
     }
 
-    for (int i = 0; i < n; ++i)
-      random_points_host(i) = {
-          {offset_x + random(), offset_y + random(), offset_z + random()}};
+    // Generate random points uniformly distributed within a box.
+    // FIXME: should be fixed to 1D/2D
+    auto const a = std::cbrt(n_values);
```

Comment:
Note this: in the latest commit this was changed from `a = std::cbrt(std::max(n_values, n_queries)`. I think this is more appropriate, and would guarantee the tree does not change with changing number of queries.